docs/GettingStarted.rst

```diff
@@ -0,0 +1,86 @@
+.. raw:: html
+
+   <!-- This Source Code Form is subject to the terms of the Mozilla Public
+      - License, v. 2.0. If a copy of the MPL was not distributed with this
+      - file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+===============
+Getting Started
+===============
+Get started by installing Frost and running some pre-built tests.
```

Comment:
What flavor of "getting started" are you intending? Getting started using or developing or ???

The reason I ask is that "using" is covered fairly well in the repo's README file. There's a bunch of new stuff there about using the `frost` command. aiui, that's the path forward for usage. (I'll try to add the readme to these docs -- it's a trick as the readme is markdown, and the docs are rsT.)

I'll assume the intent of this doc is "getting started developing". So adding that ("developing") to the title will also make that distinction clear in the table of contents.